research-prototype/Programs/Source/ipae2e.mpc

```diff
@@ -62,34 +62,21 @@ helperbits.assign_vector(
 
 helperbit_and_istrigger_flags = Array(numrows, sint)
 helperbit_and_istrigger_flags.assign_vector(
-    helperbits.get_vector(size=numrows - 1)
-    * reports.get_column(Columns.IS_TRIGGER).get_vector(size=numrows - 1),
```

Comment:
isn't the `helperbit` supposed to tell you if the preceding row is a is a trigger event?